image.yaml

```diff
@@ -148,7 +148,7 @@ modules:
           - name: cct_module
             git:
                   url: https://github.com/jboss-openshift/cct_module.git
-                  ref: master
+                  ref: sprint14
```

Comment:
It's [sprint-14](https://github.com/jboss-openshift/cct_module/tree/sprint-14) (missing hyphen).